README.md

```diff
@@ -128,15 +154,13 @@ These are the available values for the `--orgs` argument:
 This example shows how to specify that the MCP tools run against your default org when you configure the MCP server for VS Code:
 
 ```json
-     "mcp": {
        "servers": {
          "Salesforce DX": {
            "type": "stdio",
            "command": "npx",
            "args": ["-y", "@salesforce/mcp", "--orgs", "DEFAULT_TARGET_ORG"]
```

Comment:
```suggestion
           "args": ["-y", "@salesforce/mcp", "--orgs", "DEFAULT_TARGET_ORG,test-org@example.com"]
```
See the "This sample snippet shows ..." line below

Reply:
@iowillhoit , I'm a bit confused.  I thought setting `--orgs` to `DEFAULT_TARGET_ORG` _automatically_ picked up the default local (or global if local not set) target org in a DX project.  So `,test-org@example.com` isn't needed in this case.  Or am I misunderstanding?  This first example is meant to show the simplest way to set `--orgs`, and the next examples show more complex ways.

Reply:
Ah, that explains it!  So I think we're all good?